op-challenger/game/service.go

```diff
@@ -213,7 +213,8 @@ func (s *Service) initRollupClient(ctx context.Context, cfg *config.Config) erro
 func (s *Service) registerGameTypes(ctx context.Context, cfg *config.Config) error {
 	gameTypeRegistry := registry.NewGameTypeRegistry()
 	caller := batching.NewMultiCaller(s.l1Client.Client(), batching.DefaultBatchSize)
-	closer, err := fault.RegisterGameTypes(gameTypeRegistry, ctx, s.logger, s.metrics, cfg, s.rollupClient, s.txSender, s.factoryContract, caller)
+	cl := clock.SystemClock
```

Comment:
nit: I'd suggest setting the clock as a field on `Service` right up in `NewService` so that its available everywhere. Then update `initMonitor` to use that global version as well. We want to ensure we use a single clock instance across the application so may as well.

Reply:
Yep that makes sense! Will do